src/acl/external/kerberos_ldap_group/kerberos_ldap_group.cc

```diff
@@ -489,7 +489,7 @@ strup(char *s)
 #else
 #include <cstdlib>
 int
-main(int argc, char *const argv[])
+main(int , char *const [])
```

Comment:
```suggestion
main()
```

See (1) at https://en.cppreference.com/w/cpp/language/main_function